docs/resources/kubernetes_secret.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: "Kubernetes Secret Resource"
+Description: |-
+    Creating the Kubernetes Secret resource.
+---
+
+# Kubernetes Secret
```

Comment:
Please add a section for scope, refer other resources like - `security policy`

Reply:
added PTAL